packages/perseus/src/util/graphie.ts

```diff
@@ -1386,8 +1410,7 @@ export class Graphie {
             if (typeof transformer === "function") {
                 $.extend(processed, transformer(value));
             } else {
-                const dasherized = key
-                    // @ts-expect-error - TS2339 - Property 'replace' does not exist on type 'string | number | symbol'.
+                const dasherized = String(key)
```

Comment:
I wonder if this would not be needed if line 1407 above used `attrs.forEach()` instead of `$.each(attrs`..